### DISCUSSION TITLE
New Version 5" , problems

### DISCUSSION THREAD

**Discussion by dinosaur-lavender** [mathoudebine/turing-smart-screen-python#36] [Q&A]
Hello everybody.

My version is the new 5" inch version. I don't know if this could be causing an unexpected problem.

But even running with all dependencies correctly.

The console running in a loop (ok is expected), but it doesn't accept any STOP command ( CTRL+C or CTRL+Z ) and no DEBUG or ERROR message.
Getting locked straight without changing the display

Selected Hardware Revision B (XuanFang screen version B / flagship)
21:26:46 [DEBUG] Static COM port: COM5

The USB screen (it's just the default logo) when it's on.

or

File "..\turing-smart-screen-python-main\library\lcd_comm_rev_b.py", line 95, in Hello
    if response[0] != Command.HELLO or response[-1] != Command.HELLO:
IndexError: index out of range

**Screen:**
- Turing Smart Screen
- Version B
- 5" inches

**Code:**
- Change resolution 480x800 ( try default in file [3.5"] )
- Force Revision B
- Force COM5 ( auto always return None )

**YOU:**
- Windows 11
- Python 3.10

-------------------------------------------------- -------------------------

PS: In the normal software ( windows turing ) runs ok.

I'm very excited to be able to contribute with themes and the like, but I need this little help to at least run it once

Thanks

**Comment by mathoudebine** [comment]
Hi @dinosaur-lavender 
Thanks for raising this issue!

This program has not been tested for Turing Smart Screen 5" for now, only Turing Smart Screen 3.5" but it should be compatible with 5" if commands are the same.

But if i remember correctly, the 5" version is hardware revision A.
Which Windows program did you get for your 5" screen? Was it UsbMonitor.exe or ExtendScreen.exe?

**Comment by dinosaur-lavender** [comment]
The Windows program is UsbMonitorL.exe

**Reply by mathoudebine** [reply]
The USBMonitor.exe is for revision A hardware, so i assume UsbMonitorL is also for revision A
Can you try by setting revision A in `config.yaml` ?
